src/udev/udev-builtin-net_id.c

```diff
@@ -837,7 +843,37 @@ static int names_devicetree(sd_device *dev, const char *prefix, EventMode mode)
                 if (device_get_sysattr_value_filtered(aliases_dev, alias, &alias_path) < 0)
                         continue;
 
-                if (!path_equal(ofnode_path, alias_path))
+                /* if a ethernet-ports device node was found, look for ports inside of it and check if it
+                 * matches the ethernet alias */
+                if (eth_ports_dev) {
+                        eth_ports_alias_found = false;
+
+                        FOREACH_DEVICE_CHILD(eth_ports_dev, child) {
+                                r = sd_device_get_syspath(child, &eth_ports_child_syspath);
+                                if (r < 0)
+                                        return log_device_debug_errno(eth_ports_dev, r,
```

Comment:
Not a big matter, but better to make this not critical. That is, log the error, but continue the loop.

Reply:
Fixed, thanks!